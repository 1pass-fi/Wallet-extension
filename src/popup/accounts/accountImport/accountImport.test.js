import { render } from '@testing-library/react'
import '@testing-library/jest-dom/extend-expect'
import React from 'react'

import AccountImport from './index'

describe('Test for AccountImport component', () => {
  describe('Render without crashing', () => {
    it('renders correctly', () => {
      const { container } = render(<AccountImport />)
      expect(container).toMatchSnapshot()
    })
  })
})
