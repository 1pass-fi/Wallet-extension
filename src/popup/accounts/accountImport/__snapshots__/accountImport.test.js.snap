// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test for AccountImport component Render without crashing renders correctly 1`] = `
<div>
  <div
    class="account-import"
  >
    <div
      class="option-card"
    >
      <div
        class="card-title-wrapper"
      >
        <img
          class="card-icon"
          src="[object Object]"
        />
        <p
          class="card-title"
        >
          Import with a seed phrase
        </p>
      </div>
      <div
        class="card-description"
      >
        Import an existing wallet using a 12-word seed phrase.
      </div>
    </div>
    <div
      class="option-card"
    >
      <div
        class="card-title-wrapper"
      >
        <img
          class="card-icon"
          src="[object Object]"
        />
        <p
          class="card-title"
        >
          Upload a .JSON wallet file
        </p>
      </div>
      <div
        class="card-description"
      >
        Import an existing wallet by uploading a .JSON file.
      </div>
    </div>
    <div
      class="option-card"
    >
      <div
        class="card-title-wrapper"
      >
        <img
          class="card-icon"
          src="[object Object]"
        />
        <p
          class="card-title"
        >
          Get a new wallet
        </p>
      </div>
      <div
        class="card-description"
      >
        Start from the beginning.
      </div>
    </div>
  </div>
</div>
`;
